async-download-rest/src/main/java/org/uniprot/api/async/download/model/uniprotkb/UniProtKBDownloadJob.java

```diff
@@ -0,0 +1,48 @@
+package org.uniprot.api.async.download.model.uniprotkb;
+
+import java.time.LocalDateTime;
+
+import lombok.Builder;
+
+import org.springframework.data.redis.core.RedisHash;
+import org.uniprot.api.async.download.model.common.DownloadJob;
+import org.uniprot.api.rest.download.model.JobStatus;
+
+@RedisHash("uniprotkb")
+public class UniProtKBDownloadJob extends DownloadJob {
+
+    private static final long serialVersionUID = 1092075039555402818L;
+
+    @Builder
```

Comment:
Optional: we have added @Builder annotation in class level. if you want to keep the pattern, can you please move it to class level? This also apply for uniref and idmapping class as well